pages/upgrade-guide-2.x.md

```diff
@@ -196,6 +196,36 @@ QUnit.module( "router", {
 ```
 
 
+## Modified assertion methods
+
+A few assertion methods have been modified.
```

Comment:
Since there's only one thing listed here, can we ditch this `modified assertion methods` heading? Doesn't seem beneficial.